strax/corrections.py

```diff
@@ -75,6 +75,29 @@ def list_corrections(self):
         database = self.client[self.database_name]
         return [x['name'] for x in database.list_collections()]
 
+    def read_at(self, correction, when, limit=1):
+        """Smart logic to read corrections at given time (index), i.e by datetime index
+        :param correction: pandas.DataFrame object name in the DB (str type).
+        :return: DataFrame as read from the corrections database with time
```

Comment:
Could you also add when and limit to the doc-string? I do not know what this is suppose to be.

Reply:
good catch, done